website/docs/r/mrs_hybrid_cluster_v1.html.markdown

```diff
@@ -0,0 +1,233 @@
+---
+layout: "flexibleengine"
+page_title: "FlexibleEngine: flexibleengine_mrs_hybrid_cluster_v1"
+sidebar_current: "docs-flexibleengine-resource-mrs-hybrid-cluster-v1"
+description: |-
+  Manages resource cluster within FlexibleEngine MRS.
+---
+
+# flexibleengine\_mrs\_hybrid\_cluster\_v1
+
+Manages resource cluster within FlexibleEngine MRS.
+
+## Example Usage:  Creating a MRS hybrid cluster
+
+```hcl
+resource "flexibleengine_vpc_v1" "vpc_2" {
+  name = "terraform_provider_vpc2"
+  cidr= "192.168.0.0/16"
+}
+
+resource "flexibleengine_vpc_subnet_v1" "subnet_1" {
+  name = "flexibleengine_subnet"
+  cidr = "192.168.0.0/16"
+  gateway_ip = "192.168.0.1"
+  vpc_id = flexibleengine_vpc_v1.vpc_2.id
+}
+resource "flexibleengine_mrs_hybrid_cluster_v1" "cluster1" {
+  available_zone  = "eu-west-0a"
+  cluster_name    = "mrs-hybrid-cluster-acc"
+  cluster_version = "MRS 2.0.1"
+  cluster_admin_secret  = "Cluster@123"
+  master_node_key_pair = "KeyPair-ci"
+  vpc_id = flexibleengine_vpc_v1.vpc_2.id
+  subnet_id = flexibleengine_vpc_subnet_v1.subnet_1.id
+  component_list = ["Hadoop", "Storm", "Spark", "Hive"]
+  master_nodes {
+    node_number = 1
+    flavor = "s3.2xlarge.4.linux.mrs"
+    data_volume_type = "SATA"
+    data_volume_size = 100
+    data_volume_count = 1
+  }
+  analysis_core_nodes {
+    node_number = 1
+    flavor = "s3.xlarge.4.linux.mrs"
+    data_volume_type = "SATA"
+    data_volume_size = 100
+    data_volume_count = 1
+  }
+  streaming_core_nodes {
+    node_number = 1
+    flavor = "s3.xlarge.4.linux.mrs"
+    data_volume_type = "SATA"
+    data_volume_size = 100
+    data_volume_count = 1
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional) Cluster region information. Obtain the value from
+    Regions and Endpoints.
+
+* `available_zone` - (Required) ID or Name of an available zone. Obtain the value
+    from Regions and Endpoints.
+
+* `cluster_name` - (Required) Cluster name, which is globally unique and contains
+    only 1 to 64 letters, digits, hyphens (-), and underscores (_).
+
+* `cluster_version` - (Required) Version of the clusters. Currently, MRS 1.3.0, MRS 1.5.0,
+    MRS 1.6.3, MRS 1.8.9, and MRS 2.0.1 are supported. The latest version of MRS is used by default.
+   Currently, the latest version is MRS 2.0.1.
```

Comment:
currently, only  MRS 1.6.3, MRS 1.8.9, and MRS 2.0.1 are supported, please update.